### DISCUSSION TITLE
Getting the ModuleNotFoundError: No module named 'freppledb.myapp'

### DISCUSSION THREAD

**Discussion by MaxbyteTechnologies** [frePPLe/frepple#550] [Q&A]
Getting the ModuleNotFoundError: No module named 'freppledb.myapp' when trying to execute the command 'frepplectl migrate' and 'frepplectl makemigrations my_app'

Following the steps from 'Creating an extension app' from Developer Guide'

**Comment by mike-ivc-inform** [comment]
This error has little to do with frepple, but refers to incorrect work with the Django framework. Start checking your 'myapp' in the 'INSTALLED_APPS = ' section of the djangosettings.py file, check your application 'myapp' paths. ...
